panic: invalid memory address or nil pointer dereference

Dear @nohehf,

This tool is awesome. However while running the following:  `docker run --rm -it -v "$(pwd)/data:/data" escapetech/goctopus:latest -subdomain -introspect -suggest -f /data/input.txt -o /data/output.jsonl -w 50`

<img width="810" alt="Screenshot 2023-04-21 at 20 54 48" src="https://user-images.githubusercontent.com/15650113/234860605-5d3ef153-75f8-44f3-9ab1-c2225072e57d.png">

I tried again the exact same command and then it worked: I could not reproduce.

Was able to reproduce it. It seems to be the case on certain domains/addresses only. I'll investigate.
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x1028939b0]

goroutine 49 [running]:
github.com/Escape-Technologies/goctopus/pkg/suggestion.FingerprintFieldSuggestion({0x1400076f3b0, 0x27}, {0x102a92be0, 0x14000ffa120})
	/Users/nohehf/go/pkg/mod/github.com/!escape-!technologies/goctopus@v0.0.11/pkg/suggestion/field_suggestion.go:59 +0x110
github.com/Escape-Technologies/goctopus/pkg/endpoint.(*_endpointFingerprinter).HasFieldSuggestion(0x14000b0fdf8?)
	/Users/nohehf/go/pkg/mod/github.com/!escape-!technologies/goctopus@v0.0.11/pkg/endpoint/endpoint.go:39 +0x34
github.com/Escape-Technologies/goctopus/pkg/endpoint.fingerprintEndpoint(0x140002ed100, {0x102a94b10, 0x1400000cba0}, 0x1400014a3f0)
	/Users/nohehf/go/pkg/mod/github.com/!escape-!technologies/goctopus@v0.0.11/pkg/endpoint/fingerprint.go:47 +0x180
github.com/Escape-Technologies/goctopus/pkg/endpoint.FingerprintEndpoint(0x140002ed100)
	/Users/nohehf/go/pkg/mod/github.com/!escape-!technologies/goctopus@v0.0.11/pkg/endpoint/fingerprint.go:59 +0x144
github.com/Escape-Technologies/goctopus/pkg/goctopus.fingerprintAddress(0x140002ed100)
	/Users/nohehf/go/pkg/mod/github.com/!escape-!technologies/goctopus@v0.0.11/pkg/goctopus/fingerprint.go:36 +0x38
github.com/Escape-Technologies/goctopus/pkg/goctopus.worker(0x0?, 0x0?, 0x0?, 0x0?)
	/Users/nohehf/go/pkg/mod/github.com/!escape-!technologies/goctopus@v0.0.11/pkg/goctopus/fingerprint.go:20 +0x12c
created by github.com/Escape-Technologies/goctopus/pkg/goctopus.FingerprintAddresses
	/Users/nohehf/go/pkg/mod/github.com/!escape-!technologies/goctopus@v0.0.11/pkg/goctopus/fingerprint.go:52 +0xac
```

--- a/pkg/suggestion/field_suggestion.go
+++ b/pkg/suggestion/field_suggestion.go
@@ -56,11 +56,11 @@ func FingerprintFieldSuggestion(url string, client http.Client) bool {
 	for _, word := range *utils.Wordlist {
 		body := makePayload(word)
 		res, err := client.Post(url, body)
-		log.Debugf("Response from %v: %v", url, res.StatusCode)
 		if err != nil {
 			log.Debugf("Error from %v: %v", url, err)
 			return false
 		}
+		log.Debugf("Response from %v: %v", url, res.StatusCode)
 		if IsSuggestionResponse(res) {
 			return true
 		}

